proto/resource_manager.proto

```diff
@@ -95,21 +95,20 @@ message TokenBucketResponse {
   repeated GrantedRawResourceTokenBucket granted_resource_tokens = 3;
 }
 
-message GrantedRawResourceTokenBucket {
-  RawResourceType type = 1;
+message GrantedRUTokenBucket {
+  RequestUnitType type = 1;
   TokenBucket granted_tokens = 2;
   int64 trickle_time_ms = 3;
 }
 
-message GrantedRUTokenBucket {
-  RequestUnitType type = 1;
+message GrantedRawResourceTokenBucket {
+  RawResourceType type = 1;
   TokenBucket granted_tokens = 2;
   int64 trickle_time_ms = 3;
 }
 
 enum RequestUnitType {
```

Comment:
Do we still need this type?

Reply:
Better to keep it there in case some customers wanna distinct different workload rus if the overhead is minor.

Reply:
> Better to keep it there in case some customers wanna distinct different workload rus if the overhead is minor.

Customers can distinct different workload rus by `Consumption`. In fact the type only indicates that it will be used in token bucket.
I want keep RequestUnit mode and RawResource mode same in `TokenBucketRequest`